tests/retrieve-data-helpers/service-exchange.test.js

```diff
@@ -44,13 +47,18 @@ describe('Service Exchange', () => {
   }
 
   beforeEach(() => {
+    fhirConfig = require('../../src/config/fhir-config');
     mockPatient = 'patient-1';
     mockFhirServer = 'http://fhir-server-example.com';
     mockServiceWithPrefetch = 'http://example.com/cds-services/id-1';
     mockServiceWithoutPrefetch = 'http://example.com/cds-services/id-2';
     mockServiceNoEncoding = 'http://example.com/cds-services/id-3';
     mockServiceWithPrefetchEncoded = 'http://example.com/cds-services/id-4';
     mockHookInstance = '123';
+    mockAccessToken = {
+      access_token: 'access-token',
+      expires_in: 'now',
```

Comment:
I know this is a mock value that isn't checked, but this should be a value of seconds (eg, `"600"`). See https://tools.ietf.org/html/rfc6749#section-4.2.2

Reply:
@kpshek Thanks for catching that! Fixed